editor/filesystem_dock.cpp

```diff
@@ -3878,6 +3880,8 @@ void FileSystemDock::_update_import_dock() {
 		return;
 	}
 
+	emit_signal(SNAME("selection_changed"));
```

Comment:
It would be better to move this to where `_update_import_dock()` is called, unless there is a reason to do it like this. Updating the dock does not mean the selection changed (even though it works like that now).